### PR TITLE
Port to python3

### PR DESCRIPTION
**Pull request by ssjunnebo**
* Updated tests
* Added new tests
* Made sure it runs on both python 2 and 3
* Cleaned up code according to PEP8 using flake8
* Added some documentation

### CODE REVIEW

**Review by chuan-wang — APPROVED** [review]
Thanks for the great work!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Flowcell Parser
+
+[![Build Status](https://travis-ci.org/SciLifeLab/flowcell_parser.svg?branch=master)](https://travis-ci.org/SciLifeLab/flowcell_parser)   
+
+Python modules for parsing data from demultiplexed flowcells. The classes in `flowcell_parser.classes` and `flowcell_parser.db` are used by [TACA](https://github.com/SciLifeLab/TACA). The scripts `get_flowcell_stats.py` and `upload_flowcell_to_statusdb.py` are currently broken and no longer in use.
```
Comment (alneberg) [inline README.md:5]: I think we might as well remove them if they're broken and not used?

File: scripts/get_flowcell_stats.py [hunk 2]
```diff
@@ -17,37 +20,37 @@ def flowcell_stats(flowcell):
...
+                "Yield (Mbases)": 0,
+                "Raw Coverage": 0,
+                "% >= Q30bases": 0,
+                "Undetemined Reads": 0,
+                "% Undetermined Reads": 0}
         #in case of demultiplexing the Q30 bases is belonging to one of the samples
-        for sample_name, sample in samples_lane[1].iteritems():
-            if  sample_name == "unknown":
-                lane["Undetemined Reads"] += int( re.sub(',', '', sample["Reads"])) 
+        for sample_name, sample in six.iteritems(samples_lane[1]):
```
Comment (alneberg) [inline scripts/get_flowcell_stats.py:42]: Nice! I didn't know about this one!

File: tests/test_classes.py [hunk 3]
```diff
@@ -0,0 +1,248 @@
+import unittest
+import flowcell_parser.classes as classes
+import os
+import datetime
+
+
+class TestSampleSheetParser(unittest.TestCase):
```
Comment (alneberg) [inline tests/test_classes.py:7]: Wow! You wrote tests! 🥇

**Review by alneberg — APPROVED** [review]
Awesome!

**Pull request merged**